Description: Makes command execution in Makefile transparent
 This patch makes all compile install clean and other targets related
 command execution transparent by removing preceding @ symbol. This
 is required for a verbose output generation when DH_VERBOSE=1
 .
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Last-Update: 2012-07-29
--- a/lsw/Makefile
+++ b/lsw/Makefile
@@ -15,38 +15,36 @@ options:
 	@echo "CC       = ${CC}"
 
 .c.o:
-	@echo CC -c $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 lsw: ${OBJ}
-	@echo CC -o $@
-	@${CC} -o $@ ${OBJ} ${LDFLAGS}
+	${CC} -o $@ ${OBJ} ${LDFLAGS}
 
 clean:
 	@echo cleaning
-	@rm -f lsw ${OBJ} lsw-${VERSION}.tar.gz
+	rm -f lsw ${OBJ} lsw-${VERSION}.tar.gz
 
 dist: clean
 	@echo creating dist tarball
-	@mkdir -p lsw-${VERSION}
-	@cp -R LICENSE Makefile README config.mk lsw.1 ${SRC} lsw-${VERSION}
-	@tar -cf lsw-${VERSION}.tar lsw-${VERSION}
-	@gzip lsw-${VERSION}.tar
-	@rm -rf lsw-${VERSION}
+	mkdir -p lsw-${VERSION}
+	cp -R LICENSE Makefile README config.mk lsw.1 ${SRC} lsw-${VERSION}
+	tar -cf lsw-${VERSION}.tar lsw-${VERSION}
+	gzip lsw-${VERSION}.tar
+	rm -rf lsw-${VERSION}
 
 install: all
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/bin
-	@cp -f lsw ${DESTDIR}${PREFIX}/bin
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/lsw
-	@echo installing manual page to ${DESTDIR}${MANPREFIX}/man1/lsw.1
-	@sed "s/VERSION/${VERSION}/g" < lsw.1 > ${DESTDIR}${MANPREFIX}/man1/lsw.1
-	@chmod 644 ${DESTDIR}${MANPREFIX}/man1/lsw.1
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f lsw ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/lsw
+	echo installing manual page to ${DESTDIR}${MANPREFIX}/man1/lsw.1
+	sed "s/VERSION/${VERSION}/g" < lsw.1 > ${DESTDIR}${MANPREFIX}/man1/lsw.1
+	chmod 644 ${DESTDIR}${MANPREFIX}/man1/lsw.1
 
 uninstall:
 	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/lsw
+	rm -f ${DESTDIR}${PREFIX}/bin/lsw
 	@echo removing manual page from ${DESTDIR}${MANPREFIX}/man1
-	@rm -f ${DESTDIR}${MANPREFIX}/man1/lsw.1
+	rm -f ${DESTDIR}${MANPREFIX}/man1/lsw.1
 
 .PHONY: all options clean dist install uninstall
--- a/swarp/Makefile
+++ b/swarp/Makefile
@@ -16,36 +16,34 @@ options:
 	@echo "LD       = ${LD}"
 
 .c.o:
-	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 ${OBJ}: config.mk
 
 swarp: ${OBJ}
-	@echo LD $@
-	@${LD} -o $@ ${OBJ} ${LDFLAGS}
-	@strip $@
+	${LD} -o $@ ${OBJ} ${LDFLAGS}
+	strip $@
 
 clean:
 	@echo cleaning
-	@rm -f swarp ${OBJ} swarp-${VERSION}.tar.gz
+	rm -f swarp ${OBJ} swarp-${VERSION}.tar.gz
 
 dist: clean
 	@echo creating dist tarball
-	@mkdir -p swarp-${VERSION}
-	@cp -R LICENSE Makefile README config.mk ${SRC} swarp-${VERSION}
-	@tar -cf swarp-${VERSION}.tar swarp-${VERSION}
-	@gzip swarp-${VERSION}.tar
-	@rm -rf swarp-${VERSION}
+	mkdir -p swarp-${VERSION}
+	cp -R LICENSE Makefile README config.mk ${SRC} swarp-${VERSION}
+	tar -cf swarp-${VERSION}.tar swarp-${VERSION}
+	gzip swarp-${VERSION}.tar
+	rm -rf swarp-${VERSION}
 
 install: all
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/bin
-	@cp -f swarp ${DESTDIR}${PREFIX}/bin
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/swarp
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f swarp ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/swarp
 
 uninstall:
 	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/swarp
+	rm -f ${DESTDIR}${PREFIX}/bin/swarp
 
 .PHONY: all options clean dist install uninstall
--- a/lsx/Makefile
+++ b/lsx/Makefile
@@ -16,34 +16,32 @@ options:
 	@echo "CC       = ${CC}"
 
 .c.o:
-	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 lsx: ${OBJ}
-	@echo LD $@
-	@${CC} -o $@ ${OBJ} ${LDFLAGS}
-	@strip $@
+	${CC} -o $@ ${OBJ} ${LDFLAGS}
+	strip $@
 
 clean:
 	@echo cleaning
-	@rm -f lsx ${OBJ} lsx-${VERSION}.tar.gz
+	rm -f lsx ${OBJ} lsx-${VERSION}.tar.gz
 
 dist: clean
 	@echo creating dist tarball
-	@mkdir -p lsx-${VERSION}
-	@cp -R LICENSE Makefile README config.mk ${SRC} lsx-${VERSION}
-	@tar -cf lsx-${VERSION}.tar lsx-${VERSION}
-	@gzip lsx-${VERSION}.tar
-	@rm -rf lsx-${VERSION}
+	mkdir -p lsx-${VERSION}
+	cp -R LICENSE Makefile README config.mk ${SRC} lsx-${VERSION}
+	tar -cf lsx-${VERSION}.tar lsx-${VERSION}
+	gzip lsx-${VERSION}.tar
+	rm -rf lsx-${VERSION}
 
 install: all
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/bin
-	@cp -f lsx ${DESTDIR}${PREFIX}/bin
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/lsx
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f lsx ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/lsx
 
 uninstall:
 	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/lsx
+	rm -f ${DESTDIR}${PREFIX}/bin/lsx
 
 .PHONY: all options clean dist install uninstall
--- a/tabbed/Makefile
+++ b/tabbed/Makefile
@@ -15,46 +15,47 @@ options:
 	@echo "CC       = ${CC}"
 
 .c.o:
-	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 ${OBJ}: config.h config.mk
 
 config.h:
 	@echo creating $@ from config.def.h
-	@cp config.def.h $@
+	cp config.def.h $@
 
 tabbed: tabbed.o
-	@echo CC -o $@
-	@${CC} -o $@ tabbed.o ${LDFLAGS}
+	${CC} -o $@ tabbed.o ${LDFLAGS}
 
 clean:
 	@echo cleaning
-	@rm -f tabbed ${OBJ} tabbed-${VERSION}.tar.gz
+	rm -f tabbed ${OBJ} tabbed-${VERSION}.tar.gz
 
 dist: clean
 	@echo creating dist tarball
-	@mkdir -p tabbed-${VERSION}
-	@cp -R LICENSE Makefile README config.def.h config.mk \
+	mkdir -p tabbed-${VERSION}
+	cp -R LICENSE Makefile README config.def.h config.mk \
 		tabbed.1 arg.h ${SRC} tabbed-${VERSION}
-	@tar -cf tabbed-${VERSION}.tar tabbed-${VERSION}
-	@gzip tabbed-${VERSION}.tar
-	@rm -rf tabbed-${VERSION}
+	tar -cf tabbed-${VERSION}.tar tabbed-${VERSION}
+	gzip tabbed-${VERSION}.tar
+	rm -rf tabbed-${VERSION}
 
 install: all
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/bin
-	@cp -f tabbed ${DESTDIR}${PREFIX}/bin
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/tabbed
-	@echo installing manual page to ${DESTDIR}${MANPREFIX}/man1
-	@mkdir -p ${DESTDIR}${MANPREFIX}/man1
-	@sed "s/VERSION/${VERSION}/g" < tabbed.1 > ${DESTDIR}${MANPREFIX}/man1/tabbed.1
-	@chmod 644 ${DESTDIR}${MANPREFIX}/man1/tabbed.1
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f tabbed ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/tabbed
+	echo installing manual page to ${DESTDIR}${MANPREFIX}/man1
+	mkdir -p ${DESTDIR}${MANPREFIX}/man1
+	sed "s/VERSION/${VERSION}/g" < tabbed.1 > ${DESTDIR}${MANPREFIX}/man1/tabbed.1
+	chmod 644 ${DESTDIR}${MANPREFIX}/man1/tabbed.1
+	#debian speific
+	mv ${DESTDIR}${PREFIX}/bin/tabbed ${DESTDIR}${PREFIX}/bin/tabbed.default
+	mv ${DESTDIR}${MANPREFIX}/man1/tabbed.1 ${DESTDIR}${MANPREFIX}/man1/tabbed.default.1
 
 uninstall:
 	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/tabbed
-	@echo removing manual page from ${DESTDIR}${MANPREFIX}/man1
-	@rm -f ${DESTDIR}${MANPREFIX}/man1/tabbed.1
+	rm -f ${DESTDIR}${PREFIX}/bin/tabbed
+	echo removing manual page from ${DESTDIR}${MANPREFIX}/man1
+	rm -f ${DESTDIR}${MANPREFIX}/man1/tabbed.1
 
 .PHONY: all options clean dist install uninstall
--- a/sprop/Makefile
+++ b/sprop/Makefile
@@ -14,41 +14,39 @@ options:
 	@echo "CC       = ${CC}"
 
 .c.o:
-	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 ${OBJ}: config.mk
 
 sprop: ${OBJ}
-	@echo CC -o $@
-	@${CC} -o $@ ${OBJ} ${LDFLAGS}
+	${CC} -o $@ ${OBJ} ${LDFLAGS}
 
 clean:
 	@echo cleaning
-	@rm -f sprop ${OBJ} sprop-${VERSION}.tar.gz
+	rm -f sprop ${OBJ} sprop-${VERSION}.tar.gz
 
 dist: clean
 	@echo creating dist tarball
-	@mkdir -p sprop-${VERSION}
-	@cp -R LICENSE Makefile README config.mk sprop.1 ${SRC} sprop-${VERSION}
-	@tar -cf sprop-${VERSION}.tar sprop-${VERSION}
-	@gzip sprop-${VERSION}.tar
-	@rm -rf sprop-${VERSION}
+	mkdir -p sprop-${VERSION}
+	cp -R LICENSE Makefile README config.mk sprop.1 ${SRC} sprop-${VERSION}
+	tar -cf sprop-${VERSION}.tar sprop-${VERSION}
+	gzip sprop-${VERSION}.tar
+	rm -rf sprop-${VERSION}
 
 install: all
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/bin
-	@cp -f sprop ${DESTDIR}${PREFIX}/bin
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/sprop
-	@echo installing manual page to ${DESTDIR}${MANPREFIX}/man1
-	@mkdir -p ${DESTDIR}${MANPREFIX}/man1
-	@sed "s/VERSION/${VERSION}/g" < sprop.1 > ${DESTDIR}${MANPREFIX}/man1/sprop.1
-	@chmod 644 ${DESTDIR}${MANPREFIX}/man1/sprop.1
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f sprop ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/sprop
+	echo installing manual page to ${DESTDIR}${MANPREFIX}/man1
+	mkdir -p ${DESTDIR}${MANPREFIX}/man1
+	sed "s/VERSION/${VERSION}/g" < sprop.1 > ${DESTDIR}${MANPREFIX}/man1/sprop.1
+	chmod 644 ${DESTDIR}${MANPREFIX}/man1/sprop.1
 
 uninstall:
 	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/sprop
+	rm -f ${DESTDIR}${PREFIX}/bin/sprop
 	@echo removing manual page from ${DESTDIR}${MANPREFIX}/man1
-	@rm -f ${DESTDIR}${MANPREFIX}/man1/sprop.1
+	rm -f ${DESTDIR}${MANPREFIX}/man1/sprop.1
 
 .PHONY: all options clean dist install uninstall
--- a/ssid/Makefile
+++ b/ssid/Makefile
@@ -16,36 +16,34 @@ options:
 	@echo "LD       = ${LD}"
 
 .c.o:
-	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 ${OBJ}: config.mk
 
 ssid: ${OBJ}
-	@echo LD $@
-	@${LD} -o $@ ${OBJ} ${LDFLAGS}
-	@strip $@
+	${LD} -o $@ ${OBJ} ${LDFLAGS}
+	strip $@
 
 clean:
 	@echo cleaning
-	@rm -f ssid ${OBJ} ssid-${VERSION}.tar.gz
+	rm -f ssid ${OBJ} ssid-${VERSION}.tar.gz
 
 dist: clean
 	@echo creating dist tarball
-	@mkdir -p ssid-${VERSION}
-	@cp -R LICENSE Makefile README config.mk ${SRC} ssid-${VERSION}
-	@tar -cf ssid-${VERSION}.tar ssid-${VERSION}
-	@gzip ssid-${VERSION}.tar
-	@rm -rf ssid-${VERSION}
+	mkdir -p ssid-${VERSION}
+	cp -R LICENSE Makefile README config.mk ${SRC} ssid-${VERSION}
+	tar -cf ssid-${VERSION}.tar ssid-${VERSION}
+	gzip ssid-${VERSION}.tar
+	rm -rf ssid-${VERSION}
 
 install: all
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/bin
-	@cp -f ssid ${DESTDIR}${PREFIX}/bin
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/ssid
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f ssid ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/ssid
 
 uninstall:
 	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/ssid
+	rm -f ${DESTDIR}${PREFIX}/bin/ssid
 
 .PHONY: all options clean dist install uninstall
--- a/dmenu/Makefile
+++ b/dmenu/Makefile
@@ -15,52 +15,52 @@ options:
 	@echo "CC       = ${CC}"
 
 .c.o:
-	@echo CC -c $<
-	@${CC} -c $< ${CFLAGS}
+	${CC} -c $< ${CFLAGS}
 
 ${OBJ}: config.mk draw.h
 
 dmenu: dmenu.o draw.o
-	@echo CC -o $@
-	@${CC} -o $@ dmenu.o draw.o ${LDFLAGS}
+	${CC} -o $@ dmenu.o draw.o ${LDFLAGS}
 
 stest: stest.o
-	@echo CC -o $@
-	@${CC} -o $@ stest.o ${LDFLAGS}
+	${CC} -o $@ stest.o ${LDFLAGS}
 
 clean:
 	@echo cleaning
-	@rm -f dmenu stest ${OBJ} dmenu-${VERSION}.tar.gz
+	rm -f dmenu stest ${OBJ} dmenu-${VERSION}.tar.gz
 
 dist: clean
 	@echo creating dist tarball
-	@mkdir -p dmenu-${VERSION}
-	@cp LICENSE Makefile README config.mk dmenu.1 draw.h dmenu_run stest.1 ${SRC} dmenu-${VERSION}
-	@tar -cf dmenu-${VERSION}.tar dmenu-${VERSION}
-	@gzip dmenu-${VERSION}.tar
-	@rm -rf dmenu-${VERSION}
+	mkdir -p dmenu-${VERSION}
+	cp LICENSE Makefile README config.mk dmenu.1 draw.h dmenu_run stest.1 ${SRC} dmenu-${VERSION}
+	tar -cf dmenu-${VERSION}.tar dmenu-${VERSION}
+	gzip dmenu-${VERSION}.tar
+	rm -rf dmenu-${VERSION}
 
 install: all
 	@echo installing executables to ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/bin
-	@cp -f dmenu dmenu_run stest ${DESTDIR}${PREFIX}/bin
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/dmenu
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/dmenu_run
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/stest
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f dmenu dmenu_run stest ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/dmenu
+	chmod 755 ${DESTDIR}${PREFIX}/bin/dmenu_run
+	chmod 755 ${DESTDIR}${PREFIX}/bin/stest
 	@echo installing manual pages to ${DESTDIR}${MANPREFIX}/man1
-	@mkdir -p ${DESTDIR}${MANPREFIX}/man1
-	@sed "s/VERSION/${VERSION}/g" < dmenu.1 > ${DESTDIR}${MANPREFIX}/man1/dmenu.1
-	@sed "s/VERSION/${VERSION}/g" < stest.1 > ${DESTDIR}${MANPREFIX}/man1/stest.1
-	@chmod 644 ${DESTDIR}${MANPREFIX}/man1/dmenu.1
-	@chmod 644 ${DESTDIR}${MANPREFIX}/man1/stest.1
+	mkdir -p ${DESTDIR}${MANPREFIX}/man1
+	sed "s/VERSION/${VERSION}/g" < dmenu.1 > ${DESTDIR}${MANPREFIX}/man1/dmenu.1
+	sed "s/VERSION/${VERSION}/g" < stest.1 > ${DESTDIR}${MANPREFIX}/man1/stest.1
+	chmod 644 ${DESTDIR}${MANPREFIX}/man1/dmenu.1
+	chmod 644 ${DESTDIR}${MANPREFIX}/man1/stest.1
+	# debian specific
+	mv ${DESTDIR}${PREFIX}/bin/dmenu ${DESTDIR}/${PREFIX}/bin/dmenu.default
+	mv ${DESTDIR}${MANPREFIX}/man1/dmenu.1 ${DESTDIR}/${MANPREFIX}/man1/dmenu.default.1
 
 uninstall:
 	@echo removing executables from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/dmenu
-	@rm -f ${DESTDIR}${PREFIX}/bin/dmenu_run
-	@rm -f ${DESTDIR}${PREFIX}/bin/stest
+	rm -f ${DESTDIR}${PREFIX}/bin/dmenu
+	rm -f ${DESTDIR}${PREFIX}/bin/dmenu_run
+	rm -f ${DESTDIR}${PREFIX}/bin/stest
 	@echo removing manual page from ${DESTDIR}${MANPREFIX}/man1
-	@rm -f ${DESTDIR}${MANPREFIX}/man1/dmenu.1
-	@rm -f ${DESTDIR}${MANPREFIX}/man1/stest.1
+	rm -f ${DESTDIR}${MANPREFIX}/man1/dmenu.1
+	rm -f ${DESTDIR}${MANPREFIX}/man1/stest.1
 
 .PHONY: all options clean dist install uninstall
--- a/slock/Makefile
+++ b/slock/Makefile
@@ -15,26 +15,24 @@ options:
 	@echo "CC       = ${CC}"
 
 .c.o:
-	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 ${OBJ}: config.mk
 
 slock: ${OBJ}
-	@echo CC -o $@
-	@${CC} -o $@ ${OBJ} ${LDFLAGS}
+	${CC} -o $@ ${OBJ} ${LDFLAGS}
 
 clean:
 	@echo cleaning
-	@rm -f slock ${OBJ} slock-${VERSION}.tar.gz
+	rm -f slock ${OBJ} slock-${VERSION}.tar.gz
 
 dist: clean
 	@echo creating dist tarball
-	@mkdir -p slock-${VERSION}
-	@cp -R LICENSE Makefile README config.mk ${SRC} slock-${VERSION}
-	@tar -cf slock-${VERSION}.tar slock-${VERSION}
-	@gzip slock-${VERSION}.tar
-	@rm -rf slock-${VERSION}
+	mkdir -p slock-${VERSION}
+	cp -R LICENSE Makefile README config.mk ${SRC} slock-${VERSION}
+	tar -cf slock-${VERSION}.tar slock-${VERSION}
+	gzip slock-${VERSION}.tar
+	rm -rf slock-${VERSION}
 
 install: all
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
@@ -46,6 +44,6 @@ install: all
 
 uninstall:
 	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/slock
+	rm -f ${DESTDIR}${PREFIX}/bin/slock
 
 .PHONY: all options clean dist install uninstall
--- a/wmname/Makefile
+++ b/wmname/Makefile
@@ -15,36 +15,34 @@ options:
 	@echo "LD       = ${LD}"
 
 .c.o:
-	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 ${OBJ}: config.mk
 
 wmname: ${OBJ}
-	@echo LD $@
-	@${LD} -o $@ ${OBJ} ${LDFLAGS}
-	@strip $@
+	${LD} -o $@ ${OBJ} ${LDFLAGS}
+	strip $@
 
 clean:
 	@echo cleaning
-	@rm -f wmname ${OBJ} wmname-${VERSION}.tar.gz
+	rm -f wmname ${OBJ} wmname-${VERSION}.tar.gz
 
 dist: clean
 	@echo creating dist tarball
-	@mkdir -p wmname-${VERSION}
-	@cp -R LICENSE Makefile README config.mk ${SRC} wmname-${VERSION}
-	@tar -cf wmname-${VERSION}.tar wmname-${VERSION}
-	@gzip wmname-${VERSION}.tar
-	@rm -rf wmname-${VERSION}
+	mkdir -p wmname-${VERSION}
+	cp -R LICENSE Makefile README config.mk ${SRC} wmname-${VERSION}
+	tar -cf wmname-${VERSION}.tar wmname-${VERSION}
+	gzip wmname-${VERSION}.tar
+	rm -rf wmname-${VERSION}
 
 install: all
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/bin
-	@cp -f wmname ${DESTDIR}${PREFIX}/bin
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/wmname
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f wmname ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/wmname
 
 uninstall:
 	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/wmname
+	rm -f ${DESTDIR}${PREFIX}/bin/wmname
 
 .PHONY: all options clean dist install uninstall
--- a/sselp/Makefile
+++ b/sselp/Makefile
@@ -14,35 +14,33 @@ options:
 	@echo "CC       = ${CC}"
 
 .c.o:
-	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 ${OBJ}: config.mk
 
 sselp: ${OBJ}
-	@echo CC -o $@
-	@${CC} -o $@ ${OBJ} ${LDFLAGS}
+	${CC} -o $@ ${OBJ} ${LDFLAGS}
 
 clean:
 	@echo cleaning
-	@rm -f sselp ${OBJ} sselp-${VERSION}.tar.gz
+	rm -f sselp ${OBJ} sselp-${VERSION}.tar.gz
 
 dist: clean
 	@echo creating dist tarball
-	@mkdir -p sselp-${VERSION}
-	@cp -R LICENSE Makefile README config.mk ${SRC} sselp-${VERSION}
-	@tar -cf sselp-${VERSION}.tar sselp-${VERSION}
-	@gzip sselp-${VERSION}.tar
-	@rm -rf sselp-${VERSION}
+	mkdir -p sselp-${VERSION}
+	cp -R LICENSE Makefile README config.mk ${SRC} sselp-${VERSION}
+	tar -cf sselp-${VERSION}.tar sselp-${VERSION}
+	gzip sselp-${VERSION}.tar
+	rm -rf sselp-${VERSION}
 
 install: all
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/bin
-	@cp -f sselp ${DESTDIR}${PREFIX}/bin
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/sselp
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f sselp ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/sselp
 
 uninstall:
 	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/sselp
+	rm -f ${DESTDIR}${PREFIX}/bin/sselp
 
 .PHONY: all options clean dist install uninstall
--- a/xssstate/Makefile
+++ b/xssstate/Makefile
@@ -15,41 +15,41 @@ options:
 
 .c.o:
 	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 ${OBJ}: config.mk
 
 xssstate: xssstate.o
 	@echo CC -o $@
-	@${CC} -o $@ xssstate.o ${LDFLAGS}
+	${CC} -o $@ xssstate.o ${LDFLAGS}
 
 clean:
 	@echo cleaning
-	@rm -f xssstate ${OBJ} xssstate-${VERSION}.tar.gz
+	rm -f xssstate ${OBJ} xssstate-${VERSION}.tar.gz
 
 dist: clean
 	@echo creating dist tarball
-	@mkdir -p xssstate-${VERSION}
-	@cp -R LICENSE README Makefile config.mk xsidle.sh \
+	mkdir -p xssstate-${VERSION}
+	cp -R LICENSE README Makefile config.mk xsidle.sh \
 		xssstate.1 arg.h ${SRC} xssstate-${VERSION}
-	@tar -cf xssstate-${VERSION}.tar xssstate-${VERSION}
-	@gzip xssstate-${VERSION}.tar
-	@rm -rf xssstate-${VERSION}
+	tar -cf xssstate-${VERSION}.tar xssstate-${VERSION}
+	gzip xssstate-${VERSION}.tar
+	rm -rf xssstate-${VERSION}
 
 install: all
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/bin
-	@cp -f xssstate ${DESTDIR}${PREFIX}/bin
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/xssstate
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f xssstate ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/xssstate
 	@echo installing manual page to ${DESTDIR}${MANPREFIX}/man1
-	@mkdir -p ${DESTDIR}${MANPREFIX}/man1
-	@sed "s/VERSION/${VERSION}/g" < xssstate.1 > ${DESTDIR}${MANPREFIX}/man1/xssstate.1
-	@chmod 644 ${DESTDIR}${MANPREFIX}/man1/xssstate.1
+	mkdir -p ${DESTDIR}${MANPREFIX}/man1
+	sed "s/VERSION/${VERSION}/g" < xssstate.1 > ${DESTDIR}${MANPREFIX}/man1/xssstate.1
+	chmod 644 ${DESTDIR}${MANPREFIX}/man1/xssstate.1
 
 uninstall:
 	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/xssstate
+	rm -f ${DESTDIR}${PREFIX}/bin/xssstate
 	@echo removing manual page from ${DESTDIR}${MANPREFIX}/man1
-	@rm -f ${DESTDIR}${MANPREFIX}/man1/xssstate.1
+	rm -f ${DESTDIR}${MANPREFIX}/man1/xssstate.1
 
 .PHONY: all options clean dist install uninstall
